src/components/forms/CommunityForm.jsx

```diff
@@ -93,6 +150,9 @@ export const CommunityForm = () => {
 
     console.log("Received values of form: ", data);
   };
+  function onChange(value){
+  console.log(value);
+  }
```

Comment:
What does this do ?

Store cheyyande ?

Reply:
Function for the diocese option